TrailBlaze/Networking/Interceptor.m

```diff
@@ -40,4 +40,13 @@ + (NSArray *) sortInAscendingOrder: (NSArray *) etaDifferences {
     return [etaDifferences sortedArrayUsingDescriptors:sortDescriptors];
 }
 
++ (NSArray *) prunePoints: (NSArray *) routePoints numberOfPoints: (int) numberOfPoints{
+    NSMutableArray *prunedPoints = [[NSMutableArray alloc] init];
```

Comment:
You can use the `new` syntax here if you're just making an empty array 😄 
`NSMutableArray *prunedPoints = [NSMutableArray new];`